src/message.cc

```diff
@@ -60,6 +62,8 @@ Message::Message(RdKafka::Message *message):
       size = message->len();
       offset = message->offset();
 
+      if (message->key())
+        key = *message->key();
```

Comment:
Can you add curly braces here? I think they're used in every if statement currently and would like to be consistent.